### PR TITLE
Improve spatialite DatabaseWrapper attributes

### PR DESCRIPTION
**Pull request by filbasi**
# I have made things!

## Related issues
While working on custom DatabaseWrapper i encountered mypy error claiming that "DatabaseOperations" has no attribute "spatial_version". Fixes were achieved by adding ops type, which is similar to: https://github.com/typeddjango/django-stubs/blob/b81b1bf3868eaab9fecebdd56ed817ffcb07fad8/django-stubs/db/backends/postgresql/base.pyi#L23

### CODE REVIEW

**Review by intgr — APPROVED** [review]
Thanks!

This is already an improvement, but would be great to fix other types too in this file. E.g. `ops_class` should be `type[SpatiaLiteOperations]`

**Comment by filbasi** [comment]
Done

**Review by intgr — APPROVED** [review]

File: django-stubs/contrib/gis/db/backends/spatialite/base.pyi [hunk 1]
```diff
@@ -2,13 +2,20 @@ from typing import Any
...
+from .schema import SpatialiteSchemaEditor
+
 class DatabaseWrapper(SQLiteDatabaseWrapper):
-    SchemaEditorClass: Any
-    client_class: Any
-    features_class: Any
-    introspection_class: Any
-    ops_class: Any
-    lib_spatialite_paths: Any
+    SchemaEditorClass: SpatialiteSchemaEditor
```
Comment (intgr) [inline django-stubs/contrib/gis/db/backends/spatialite/base.pyi]: Fails in CI. I think you need `type[]` around this.

**Comment by filbasi** [comment]
Sorry about that. Now it should run fine.

**Review by intgr — APPROVED** [review]
Thanks!

**Pull request merged**